week6/Filmer/app/src/main/java/com/tahhan/filmer/repository/MovieRepo.kt

```diff
@@ -22,30 +23,21 @@ class MovieRepo(application: Application) {
     }
```

Comment:
I'd pass in the dao to the repo, instead of the application, so that I don't have to worry about creating any dependencies here!